gradle/dependencies.gradle

```diff
@@ -88,8 +88,8 @@ else
 // See https://issues.apache.org/jira/browse/KAFKA-12622 for steps to verify the LICENSE-binary file is correct.
 versions += [
   activation: "1.1.1",
-  apacheda: "1.0.2",
-  apacheds: "2.0.0-M24",
+  apacheda: "2.1.5",
+  apacheds: "2.0.0.AM27",
```

Comment:
your PR description mentions that you removed apacheDS but we seem to be using it. Does the description need an update?

Reply:
oh thanks comment
When I first implemented it, it seemed like it could be removed, but I think I need to use some features.
Updated PR description!